includes/views/attachment-settings-hint.php

```diff
@@ -19,6 +19,7 @@
 </style>
 <!-- <div id="message" class="updated woocommerce-message"> -->
 <div class="wcpdf-attachment-settings-hint">
-	<?php printf(__( "It looks like you haven't setup any email attachments yet, check the settings under <b>%sDocuments > Invoice%s</b>", 'woocommerce-pdf-invoices-packing-slips' ), '<a href="'.$invoice_settings_url.'">', '</a>'); ?><br>
+<?php /* translators: <a> tags */ ?>
+	<?php printf(__( 'It looks like you haven\'t setup any email attachments yet, check the settings under <b>%1$sDocuments > Invoice%2$s</b>', 'woocommerce-pdf-invoices-packing-slips' ), '<a href="'.$invoice_settings_url.'">', '</a>'); ?><br>
```

Comment:
I prefer using the double quotes if there's a single quote in the text TBH

Reply:
The issue in this case is that the numbered placeholders uses the dollar sign. Do you prefer using double quotes and escape the dollar sign instead?

Reply:
I see, I didn't consider that. Let's leave it as you have it now then.